### PR TITLE
TC 04.3.4_01 <Men page> Page contains Shop by category block which is located on the left side of the page

### PR DESCRIPTION
**Pull request by yauheniumetski**
https://trello.com/c/cr8T7B8T/297-tc-043401-men-page-page-contains-sort-by-category-block-which-is-located-on-the-left-side-of-the-page

### CODE REVIEW

**Comment by yauheniumetski** [comment]
[TC 04.3.4_01 <Men page> Page contains Shop by category block which is located on the left side of the page](https://trello.com/c/cr8T7B8T/297-tc-043401-men-page-page-contains-shop-by-category-block-which-is-located-on-the-left-side-of-the-page)

**Comment by yauheniumetski** [comment]
Cards for the skipped tests are created:
https://trello.com/c/cLhj8qKj/392-err-menu-menspec-verify-the-display-of-my-wish-list-on-the-mens-page
https://trello.com/c/fh7NYRyA/408-err-productcardwishlistspec-product-card-add-to-wish-list-validate-the-message-the-product-has-been-moved-to-your-wish-list

File: tests/productCard_WishList.spec.js [hunk 1]
```diff
@@ -2,40 +2,40 @@ import { test, expect } from "@playwright/test";
...
+    await page.waitForTimeout(1000);
+    await page.locator('.action.viewcart').click();
 
-        await expect(page.locator('.page.messages')).toBeVisible;
-        await expect(page.locator('.page.messages')).toContainText('Inez Full Zip Jacket has been moved to your wish list.');
-        });
+    await expect(page.getByText('Move to Wishlist')).toBeVisible();
+  });
+
+  test.skip("Validate the message - the product has been moved to your wish list", async ({ page }) => {
```
[the remark this answers is not in the document]
Comment (yauheniumetski) [reply tests/productCard_WishList.spec.js:31]: > The rule of our project is 1 PR - 1 test please
@ElNov686 
Thanks. I know about the rule. On the my comment bellow I've created 2 cards with errors which are blocking whole tests for PR. These tests are created by other team members. I've made it according to last video: 13. (04/18) Ревью тестов, карточки Refactoring, ERROR. Trace Viewer (Maria Moroz). Please review my PR again

**Review by BabintsevaS — COMMENTED** [review]
name in the test.describe should be the same as file name - "menPage"

**Comment by yauheniumetski** [comment]
> menPage

@BabintsevaS Fixed

**Review by Chesnova — APPROVED** [review]

**Review by BabintsevaS — APPROVED** [review]

**Pull request merged**